db/initsql/dbschema.sql

```diff
@@ -0,0 +1,74 @@
+create table if not exists room
+(
+	id integer not null
+		constraint room_pkey
+			primary key,
+	name varchar(25) not null,
+	room_type varchar(6) not null
+);
+
+alter table room owner to postgres;
+
+create table if not exists users
+(
+	id integer not null
+		constraint users_pkey
+			primary key,
+	name varchar(20) not null,
+	surname varchar(20) not null
+);
+
+alter table users owner to postgres;
+
+create table if not exists movie
+(
+	id integer not null
+		constraint movie_pkey
+			primary key,
+	description varchar(1000),
+	premier_end date not null,
```

Comment:
a jak nie wiemy ile czasu będziemy grali film w kinie to go tu nie wpiszemy?
czy nazwa `premier` na pewno oddaje to co chcesz?

Reply:
> a jak nie wiemy ile czasu będziemy grali film w kinie to go tu nie wpiszemy?
> czy nazwa `premier` na pewno oddaje to co chcesz?

tu raczej pytanie biznesowe czy jest to pole potrzebne i czy ma to być wymagalne. Można to zmienić jesli będą takie zasady